README.md

```diff
@@ -34,7 +37,8 @@ The same environment is configured for GitHub Codespaces via devcontainer config
 ### env variables
 
 * if you pass `CODETRACER_RUBY_TRACER_DEBUG=1`, you enables some additional debug-related logging
-* `CODETRACER_DB_TRACE_PATH` can be used to override the path to `trace.json` (it's used internally by codetracer as well)
+* `CODETRACER_RUBY_RECORDER_OUT_DIR` can be used to specify the directory for trace files
+(used internally by codetracer as well; defaults to the current directory)
```

Comment:
it's not used internally: is this connected to a codetracer PR?

Reply:
It was renamed to establish consistency with the CLI option name and to define a convention that we can use in all future recorder projects.

Reply:
makes sense. a single variable is easier to maintain, but maybe this way it's more recorder/user-friendly indeed